resources/splunk_install.rb

```diff
@@ -60,25 +50,41 @@ def package_path
 
   load_current_value do |desired|
     package desired.package
-    version_file = version_pathname(install_dir)
-    current_value_does_not_exist! unless version_file.exist?
 
-    version_data = version_file.read
-    installed_version, installed_build = version_data.match(/VERSION=(\d(?:\.\d)+).+BUILD=([\w]+)/m).captures
-    version installed_version
-    build installed_build
+    user_resource = Chef::Resource::User.new(desired.user || new_resource.user, run_context)
```

Comment:
Doesn't this resource have to be added to the resource collection at some point? I'm missing the connection where this actually gets executed.